lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -0,0 +1,365 @@
+defmodule Mix.Tasks.Release.Init do
+  use Mix.Task
+
+  @shortdoc "Generates sample files for releases"
+
+  @moduledoc """
+  Generates sample files for releases.
+
+      mix release.init
+      * creating rel/vm.args.eex
+      * creating rel/start.eex
+      * creating rel/start.bat.eex
+
+  """
+
+  import Mix.Generator
+
+  @switches [
+    force: :boolean,
+    quiet: :boolean
+  ]
+
+  @aliases [
+    f: :force
+  ]
+
+  def run(args) do
+    {opts, args} = OptionParser.parse!(args, strict: @switches, aliases: @aliases)
+
+    if args != [] do
+      Mix.raise("Expected \"mix release.init\" without arguments, got: #{inspect(args)}")
+    end
+
+    create_file("rel/vm.args.eex", vm_args_text(), opts)
+    create_file("rel/start.eex", start_text(), opts)
+    create_file("rel/start.bat.eex", start_bat_text(), opts)
+    :ok
+  end
+
+  @doc false
+  def vm_args_text,
+    do: ~S"""
+    ## Preloads all modules instead of loading them dynamically
+    -mode embedded
+
+    ## Number of dirty schedulers doing IO work (file, sockets, etc)
+    ##+SDio 5
+
+    ## Increase number of concurrent ports/sockets
+    ##-env ERL_MAX_PORTS 4096
```

Comment:
That’s actually less than the default nowadays:

http://erlang.org/doc/man/erl.html#max_ports

http://erlang.org/doc/efficiency_guide/advanced.html#system-limits